rhel8/kickstart/ssg-rhel8-ospp-ks.cfg

```diff
@@ -0,0 +1,177 @@
+# SCAP Security Guide OSPP/USGCB profile kickstart for Red Hat Enterprise Linux 8 Server
```

Comment:
Should `Server` be removed from all kickstart files as it isn't a variant anymore?